### DISCUSSION TITLE
Adding a style at the end of generated CSS

### DISCUSSION THREAD

**Discussion by jayhlee** [tailwindlabs/tailwindcss#4346] [Help] [answered]
I'm trying to add a style at the very end of my CSS that's generated by JIT. Is there a way to control where a style is outputted?

I'm trying to add `html{ visibility: hidden;opacity:0; }` at the very end of my CSS to prevent FOUC.

**Comment by simonswiss** [comment, marked as answer]
Hey!

You can add the styles in your CSS files where you have the three Tailwind directives. 

You'll need to add the [`@tailwind screens` directive](https://tailwindcss.com/docs/functions-and-directives#tailwind) as well if you want your CSS to be the absolute last thing to be generated, like so:

```
@tailwind base;
@tailwind components;
@tailwind utilities;

@tailwind screens;

/* Your custom CSS after everything else here */
.foo {
  font-size: 3rem;
}
```

Hope it helps! 👍

**Reply by jayhlee** [reply]
Thanks, Simon!
